TACC/Core-Portal
# task/FP-906: Clear search bar in Data Files when changing systems.

## Overview: ##

## Related Jira tickets: ##

* [FP-906](https://jira.tacc.utexas.edu/browse/FP-906)
* 
## Testing Steps: ##
1. Run a search in Data Files
2. Navigate to a different system in Data Files
3. Confirm that the query in the search bar has been removed.
